handbook/business-operations/ceo-handbook.md

```diff
@@ -111,17 +111,30 @@ Every month Charlie will do the prep work for the monthly "✌️ All hands 🖐
 
 The day before the All Hands, Mike will prepare slides that reflect the CEO vision and focus. 
 
+### Post all hands
+
+The EA / CEO will post the link to the All hands meeting in #general slack channel
+Template to use:
+Thanks to everyone who contributed to today's "All hands" call.
+
+:tv: If you weren't able to attend, please **watch the recording** _(1.5x playback supported)_.
+
+You can also grab a copy of the [original slides](https://fleetdm.com/handbook/business-operations#all-hands)
```

Comment:
^wrap this template in a code block

Reply:
https://www.markdownguide.org/cheat-sheet/#extended-syntax

Reply:
To upload a screenshot, you can paste an image into a comment block.  So copy an image, and then paste the image into a github comment box.

![screenshot of how to share Gong call](https://github.com/fleetdm/confidential/assets/618009/053dcdfa-82ec-45e0-a393-0eca52bba160)